azext_iot/tests/iothub/configurations/test_iot_config_int.py

```diff
@@ -294,6 +308,53 @@ def test_edge_deployments(self):
                 ],
             )
 
+            # Edge billable module deployment
+            if auth_phase == AuthenticationTypeDataplane.login.value:
```

Comment:
Alternatively `--auth-type login` can be statically set for AAD only commands making a simpler test (reduces branching) with a small cost to redundancy.